src/tests/ftest/util/ior_test_base.py

```diff
@@ -385,3 +388,33 @@ def verify_pool_size(self, original_pool_info, processes):
             self.fail(
                 "Pool Free Size did not match: actual={}, expected={}".format(
                     actual_pool_size, expected_pool_size))
+
+    def execute_cmd(self, cmd, fail_on_err=True, display_output=True):
```

Comment:
Can't we use ExecutableCommand class for this purpose ?

Reply:
I have added execute_cmd inside ior_test_base as it is also used by some other PRs in flight using IOR. Also using ior_test_base seems more apt to me saving a lot of duplicate code of pool create, container create, dfuse mount using those pool and container and now this execute_cmd method.